### PR TITLE
GFM-159 - updated to use hof-bootstrap

### PR DESCRIPTION
**Pull request by joefitter**
* removed unnec deps
* removed setup from app.js
* installed hof-bootstrap
* added bootstrap setup to app.js
* removed apps/gro/index.js setup
* moved steps config to apps/gro/index.js

### CODE REVIEW

**Comment by JoeChapman** [comment]
Build failed with `Error: Cannot find module 'nodemailer'`

File: config.js [hunk 1]
```diff
@@ -3,21 +3,8 @@
...
-  listen_host: process.env.LISTEN_HOST || '0.0.0.0',
-  session: {
-    secret: process.env.SESSION_SECRET || 'howdoesyourgardengrow',
-    ttl: process.env.SESSION_TTL || 1800 /* 30 mins timeout */
-  },
-  redis: {
-    port: process.env.REDIS_PORT || '6379',
-    host: process.env.REDIS_HOST || '127.0.0.1'
-  },
+  env: process.env.NODE_ENV,
```
Comment (JoeChapman) [inline config.js]: isn't this covered in the bootstrap options?

**Comment by JoeChapman** [comment]
LGTM

File: config.js (hunk 1 above)
Comment (joefitter) [inline config.js]: it is internal to bootstrap - we need to know the env in the app to apply the mock postcode middleware. If we expose bootstrap config we would have already called bootstrap so would be too late

**Comment by joefitter** [comment]
acceptance tests not connecting to app it looks like - may need some drone config tweaks - looking now

**Pull request merged**